balrogscript/requirements/local.in

```diff
@@ -0,0 +1,3 @@
+-r test.in
```

Comment:
For reasons I don't fully understand, this isn't needed. When I run:
```
$ pip-compile-multi -g base -g test -o py38.txt
$ pip-compile-multi -g base -g test
```

I get hashes in all of the files. Maybe it's because `local.in` references another file that we have enabled hashes for?